ui/appui-abstract/src/appui-abstract/statusbar/StatusBarItem.ts

```diff
@@ -44,7 +44,7 @@ export enum StatusBarLabelSide {
 }
 
 /** Type for StatusBar Item Id
- * @deprecated in 3.6.
+ * @deprecated in 3.6. Please use StatusBarItemId from @itwin/appui-react.
```

Comment:
I was planning to get rid of this altogether.
```suggestion
 * @deprecated in 3.6. Please use `CommonStatusBarItem["id"]` from @itwin/appui-react.
```